src/hotspot/share/runtime/arguments.cpp

```diff
@@ -1041,6 +1043,10 @@ void Arguments::build_jvm_flags(const char* arg) {
   add_string(&_jvm_flags_array, &_num_jvm_flags, arg);
 }
 
+void Arguments::build_jvm_restore_flags(const char* arg) {
```

Comment:
I would use `append_jvm_restore_flags` - `build` implies the final operation on a builder.

Reply:
I follow the style of the class: there are `Arguments::build_jvm_args` and `Arguments::build_jvm_flags` which do exactly the same thing, just for two different arrays.